### ISSUE TITLE
MAL and hummingbird integration?

### ISSUE THREAD

**Issue by irockdude554** [CindrInc/Kami-Desktop-Client#4]
do you plan on added the ability to integrate your hummingbird or myanimelist account into this desktop app?

**Comment by IshanArya** [comment]
Yes. That's the plan after putting more sources. Just for clarification, what does MAL/Kitsu allow you to do? What type of functionality do you want from it?

**Comment by alaadotsol** [comment]
It allows you to save the shows/episodes you've seen (Like Netflix). If _ per example _ I watched ep #12 in an anime it would automatically check it as "watched" in my MAL profile.
